src/models/problem.ts

```diff
@@ -7,6 +7,11 @@ export const ProblemModel = types
     title: types.string,
     subtitle: "",
     sections: types.array(SectionModel)
+  })
+  .actions((self) => {
```

Comment:
This should be in `views` since it doesn't change anything.

Reply:
Right, that was a brain fart.